examples/JsEventHandlerExamples/multi-field-manipulation.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Name: Field Manipulation Event Handlers
+ * Description: This is a set of event handlers that demonstrate the various methods to manipulate fields.
+ */
+(function (eventNames, convenienceApi) {
+	var eventHandlers = {};
+
+	// https://platform.relativity.com/RelativityOne/Content/FormsAPI/Load_Pipeline.htm#transformlayout
+	eventHandlers[eventNames.TRANSFORM_LAYOUT] = function(layoutData) {
+		console.log("Inside TRANSFORM_LAYOUT event handler");
+		console.log(JSON.stringify(layoutData));
+		layoutData.splice(3, 1); //assuming the category you want to remove is at the bottome of the layout, you can remove it like this.
+	};
+
+	// https://platform.relativity.com/RelativityOne/Content/FormsAPI/Load_Pipeline.htm#hydratelayout
+	eventHandlers[eventNames.HYDRATE_LAYOUT] = function(layoutData, objectInstanceData) {
+        console.log("Inside HYDRATE_LAYOUT event handler");
+        console.log(JSON.stringify(objectInstanceData));
+		// Depending on if you need the previously saved values of the hidden fields, you can retrieve them out of this object instance data object.
+    };
+
+	eventHandlers[eventNames.PAGE_INTERACTION] = function() { 
+		console.log( "Inside PAGE_INTERACTION event handler" ); 
```

Comment:
should we include a commented link to the pageInteraction documentation like the other events?